### PR TITLE
Solucionado el issue 30. Ahora a los superusers les aparece un boton de ...

### PR DESCRIPTION
**Pull request by bbotella**
...delete en cada mencion en la vista index.
# Issue30

### CODE REVIEW

**Comment by nukeador** [comment]
Maybe we'll need some kind of confirmation to avoid accidental deletions.

**Comment by bbotella** [comment]
Sure. And, by the way, is it ok if the link is just available to superusers? Should we check if they are just in the "admin" group?

**\* Sorry for the spanish comments. I'm just a newie.

**Comment by bbotella** [comment]
It has been made following admin style.

File: mozbuzz/buzz/templates/buzz/delete_mention.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}{% load i18n query %}
+
+{% block main_content %}
+    <h1>
+        Are you sure?
```
Comment (nukeador) [inline mozbuzz/buzz/templates/buzz/delete_mention.html]: Strings are not l10n friendly.

Comment (bbotella) [inline mozbuzz/buzz/templates/buzz/delete_mention.html]: Uops. My bad. Fixed. It won't happen again.

File: mozbuzz/buzz/templates/buzz/index.html [hunk 2]
```diff
@@ -80,6 +80,9 @@
                     {% if user.is_authenticated %}
                         <li><a href="{% url "edit" mention.id %}">Edit</a></li>
                     {% endif %}
+                    {% if user.is_superuser %}
+                        <li><a href="{% url "delete" mention.id %}">Delete</a></li>
```
Comment (nukeador) [inline mozbuzz/buzz/templates/buzz/index.html]: We could take the chance and fix this page strings to be all l10n ready.

Comment (bbotella) [inline mozbuzz/buzz/templates/buzz/index.html]: I think I did them all.

**Comment by nukeador** [comment]
r? @sebasmagri 

@bbotella It would be helpful if once this is reviewed you can squash all commit in one:

https://help.github.com/articles/interactive-rebase

And the commit message to follow this format:

"Delete button for admins. Fix #30"

http://mhdev.readthedocs.org/es/latest/git.html#commit-messages

Thanks.

**Comment by sebasmagri** [comment]
I've been following this and it looks ok in general. @bbotella please consider @nukeador's suggestions to merge the changes.

Thanks a lot.

Regards,

**Pull request merged**